docs/walkthroughs/local-setup.adoc

```diff
@@ -110,21 +110,132 @@ as part of your `ansible-playbook` command.
 For developing the OpenShift UI locally, see link:https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/developing-ui-locally.adoc[this guide].
```

Comment:
We don't need this, the document is user facing no local dev

Reply:
oh I see it under local-setup. I think this should be a separate doc explaining to developers what the mobile custom resource is and what we use it for (representing mobile clients in OpenShift, show platform specific information and docs, binding to mobile services so that you can get the config etc) and then also the technichal information below on how to actually create the client representation